package.json

```diff
@@ -22,8 +22,10 @@
 		"element-ready": "^3.0.0",
 		"github-injection": "^1.0.1",
 		"github-reserved-names": "^1.0.14",
+		"graphql-request": "^1.6.0",
```

Comment:
As @bfred-it mentioned, we don't need this much abstraction. The `graph.js` file is fine IMO.

Reply:
removed already with lodash from the new push